DrugResistance/src/main/java/edu/stanford/hivdb/drugresistance/database/ConditionalComments.java

```diff
@@ -54,8 +54,8 @@
 		final private ConditionType conditionType;
 		final private Map<String, Object> conditionValue;
 		final private String comment;
-
-		private ConditionalComment(
+		
+		protected ConditionalComment(
```

Comment:
I promoted this class to `protected` so I could construct and test ConditionalComment instances with specificity.  However, it seems that instances of this class should be accessed via `getComments()`. So I can make this class private again and re-write these tests in terms of `getComment()` if we should keep this class's constructor private.